scripts/Mansion.gd

```diff
@@ -1503,6 +1508,11 @@ func _on_end_pressed():
 	get_node("Navigation/endlog").disabled = false
 	nextdayevents()
 
+	globals.state.daily_reports.global = text0.get_bbcode()
+	globals.state.daily_reports.job = text1.get_bbcode()
+	globals.state.daily_reports.secondary = text2.get_bbcode()
+	globals.state.daily_reports.farm = text3.get_bbcode()
```

Comment:
My main concern here is order of operations. I think this needs to be above nextdayevents(), but we'll test it as is and confirm.

Reply:
You're right about this - though for some reason, it still works a lot of the time. I think the daily event processing can delay the save, so this is allowed to finish before saving. But when there's no events, it will save the previous days text.

Will make a new PR moving to just before the autosave in startnewday, since it looks like some events can also add text to the report.